### PR TITLE
Update netdata commands

### PR DESCRIPTION
**Pull request by tkatsoulas**
##### Summary

Adding the `-W buildinfo` options so users could be aware of this option

##### Test Plan

CI tests

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 

1. improvements area/docs

2. User will be aware about their Agent build details. 

</details>

### CODE REVIEW

File: daemon/README.md [hunk 1]
```diff
@@ -180,6 +180,14 @@ The command line options of the Netdata 1.10.0 version are the following:
   -W set section option value
                            set netdata.conf option from the command line.
 
+  -W buildinfo             Print it's version, the configure options, 
+                           and a list of optional features and whether they 
+                           are enabled or not in human readable form.
+
+  -W buildinfojson         Print it's version, the configure options, 
+                           and a list of optional features and whether they 
+                           are enabled or not in json format.
```
Comment (ilyam8) [inline daemon/README.md:189]:
```suggestion
                           are enabled or not in JSON format.
```

**Comment by tkatsoulas** [comment]
thanks @ilyam8

**Comment by tkatsoulas** [comment]
On a second thought, we need to also include this info in https://github.com/netdata/netdata/blob/master/daemon/main.c#L350-L352

**Review by ilyam8 — APPROVED** [review]

**Review by thiagoftsm — APPROVED** [review]
Help is print correctly and we do not have warnings while we are compiling, LGTM!

**Pull request merged**